dev/NumberBox/InteractionTests/NumberBoxTests.cs

```diff
@@ -245,13 +245,94 @@ public void CoersionTest()
             }
         }
 
+        [TestMethod]
+        public void BasicCalculationTest()
+        {
+            using (var setup = new TestSetupHelper("NumberBox Tests"))
+            {
+                RangeValueSpinner numBox = FindElement.ByName<RangeValueSpinner>("TestNumberBox");
+
+                Log.Comment("Verify that calculations don't work if AcceptsCalculations is false");
+                EnterText(numBox, "5 + 3");
+                Verify.AreEqual(0, numBox.Value);
+
+                Check("CalculationCheckBox");
+
+                int numErrors = 0;
+                const double resetValue = 1234;
+                
+                Dictionary<string, double> expressions = new Dictionary<string, double>
+                {
+                    // Valid expressions. None of these should evaluate to the reset value.
```

Comment:
>// Valid expressions. None of these should evaluate to the reset value. [](start = 20, length = 71)

I'd like to see a couple more starts with negative and 0 expressions in the test cases

Reply:
Can you suggest some? I tried to be pretty thorough but am definitely not sure I thought of everything.